app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -65,8 +66,17 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
     showProfile = profileClick;
     showSettings = settingsClick;
 
-    creatorDashboardButtonIsGone = environment.currentConfig().observable()
-      .map(config -> !coalesce(config.features().get(FeatureKey.ANDROID_CREATOR_VIEW), false));
+    final Observable<Boolean> userIsCreator = currentUser.observable()
+      .map(u -> u != null && u.createdProjectsCount() > 0);
+
+    final Observable<Boolean> userCanViewCreatorDash = environment.currentConfig().observable()
```

Comment:
to be even a lil more explicit i'd name this something like `creatorViewFeatureFlagIsEnabled`, that way it'd take even less time for someone not familiar with this logic to know where the boolean value is coming from